ansible/tests/unit/v2/values-secret-v2-base.yaml

```diff
@@ -0,0 +1,42 @@
+# Needed to specify the new format (missing version means old version: 1.0 by default)
+version: 2.0
+# NEVER COMMIT THESE VALUES TO GIT
```

Comment:
Is this true anymore?

Reply:
Sometimes? I mean it is true unless the secret file is based on generate or prompt *only*, so I think it is best practice to just not commit it to git

Reply:
I removed it from the unit test files, but we will want something in the official examples or in the docs telling the user that in general it is best not to commit to git *unless* all the passwords are generated